vhdbuilder/packer/vhd-image-builder-mariner-arm64.json

```diff
@@ -30,6 +34,7 @@
       "type": "azure-arm",
       "virtual_network_name": "{{user `vnet_name`}}",
       "virtual_network_subnet_name": "{{user `subnet_name`}}",
+      "ssh_read_write_timeout": "5m",
```

Comment:
hmm, what's this for?

Reply:
I noticed this in the ubuntu template, and saw that it was adding some resilience to the reboot step
_Sources:_
 ([link](https://developer.hashicorp.com/packer/plugins/builders/azure/arm#ssh_read_write_timeout)) packer docs
([link](https://discuss.hashicorp.com/t/how-to-reboot-vm-with-packer/46083)) forum post on packer reboots

Reply:
yep this is something I initially added